module/VuFind/src/VuFind/View/Helper/Root/Developer.php

```diff
@@ -0,0 +1,80 @@
+<?php
+
+/**
+ * Developer helper
+ *
+ * PHP version 8
+ *
+ * Copyright (C) Villanova University 2025.
+ *
+ * This program is free software; you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License version 2,
+ * as published by the Free Software Foundation.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program; if not, write to the Free Software
+ * Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301  USA
+ *
+ * @category VuFind
+ * @package  View_Helpers
+ * @author   Juha Luoma <juha.luoma@helsinki.fi>
+ * @license  http://opensource.org/licenses/gpl-2.0.php GNU General Public License
+ * @link     http://vufind.org/wiki/ Wiki
+ */
+
+namespace VuFind\View\Helper\Root;
+
+use Laminas\View\Helper\AbstractHelper;
+use VuFind\Db\Entity\UserEntityInterface;
+
+/**
+ * Developer helper
+ *
+ * @category VuFind
+ * @package  View_Helpers
+ * @author   Juha Luoma <juha.luoma@helsinki.fi>
+ * @license  http://opensource.org/licenses/gpl-2.0.php GNU General Public License
+ * @link     http://vufind.org/wiki/ Wiki
+ */
+class Developer extends AbstractHelper
+{
+    /**
+     * Constructor
+     *
+     * @param ?UserEntityInterface $user Current logged in user or null
+     */
+    public function __construct(
+        protected ?UserEntityInterface $user = null
```

Comment:
We no longer use the $user property. Should we drop it and simplify the code?

Reply:
Yeah, removed the user and factory also as it wasnt necessary for now.